tests/defences/preprocessor/test_jpeg_compression.py

```diff
@@ -103,13 +103,14 @@ def test_jpeg_compression_video_data(art_warning, video_batch, channels_first):
 def test_jpeg_compress(art_warning, image_batch, channels_first):
     try:
         test_input, test_output = image_batch
-        jpeg_compression = JpegCompression(clip_values=(0, 255))
+        if test_input.shape[-1] in [1, 3]:
```

Comment:
Should we maybe add a short comment to explain why we only need to test two cases here (one RGG and one grayscale)? And thus skip the image batches with two and 5 channels?

Reply:
That's good idea, added with ccca32d9652c1f2df4d28fc984975f79fd3166a5